command/ssh/config.go

```diff
@@ -73,6 +73,11 @@ $ step ssh config --set User=joe --set Bastion=bastion.example.com
 				Name:  "host",
 				Usage: `Configures a SSH server instead of a client.`,
 			},
+			cli.StringFlag{
+				Name: "endpoint",
```

Comment:
I think something more specific would be better `--team-endpoint` or `--team-url`

Reply:
Probably the latter, as you're asking for a url.